### ISSUE TITLE
[Feature] Raw filter

### ISSUE THREAD

**Issue by stoffern** [nodkz/graphql-compose#13]
The posibility to add a custom raw filter to the query.
`resolveParams.args.rawFilter`

`rawQueryHelper(resolveParams)`
```js
resolve: (resolveParams : ExtendedResolveParams) => {
  resolveParams.query = model.find();
  rawQueryHelper(resolveParams);
  filterHelper(resolveParams);
  skipHelper(resolveParams);
  limitHelper(resolveParams);
  sortHelper(resolveParams);
  projectionHelper(resolveParams);
  return resolveParams.query.exec();
},
```

if so would possible need fixes in the relay and connections repo?

**Comment by nodkz** [comment]
By GraphQL spec raw queries are not recommended (they didn't have types). All input data should be validated. Also GraphQL args (InputObjectTypes) do not support union types yet, which can allow use build in validation.

Anyway we can provide such option (which can be explicitly enabled).

Needs good proposal for this feature. Also needs keep in mind, that this feature should support any future plugins (eg. sequilize).

**Comment by nodkz** [comment]
Relay and connection repos should work out of the box.

Connection requires valid findMany resolver. It can has any complexity and functionality.
Relay wraps all resolver args and put them to `input` arg. So resolvers may have any args, Compose-Relay try find `input` arg, if found do nothing; if not found arg with `input` name, then create it and move all args into this arg.

**Comment by nodkz** [comment]
Closed due new `filter args proposal` in #22
